Update signing-ios.md (#495)
codemagic-ci-cd/codemagic-docs: content/yaml-code-signing/signing-ios.md

- `APP_STORE_CONNECT_PRIVATE_KEY`

  This is the private API key downloaded from App Store Connect.

- `CERTIFICATE_PRIVATE_KEY`

>   An RSA 2048 bit private key to be included in the [signing certificate](https://help.apple.com/xcode/mac/current/#/dev1c7c2c67d) that Codemagic creates. You can use an existing key or create a new 2048 bit RSA key by running the command below in your terminal. Running the command line will create private and public keys. Note that you should click **secure** to encrypt the **contents** of the private file in the **Environment Variables** Section. 

Reviewer: ```suggestion
  An RSA 2048 bit private key to be included in the [signing certificate](https://help.apple.com/xcode/mac/current/#/dev1c7c2c67d) that Codemagic creates. You can use an existing key or create a new 2048 bit RSA key by running the command below in your terminal. Running the command line will create private and public keys. Note that you should select **Secure** to encrypt the **contents** of the private file in the **Environment Variables** Section. 
```

```bash
ssh-keygen -t rsa -b 2048 -m PEM -f ~/Desktop/codemagic_private_key -q -N ""
```

{{<notebox>}}

> Tip: Store all the Apple store variables in the same group so they can be imported to codemagic.yaml workflow at once. 

Reviewer: ```suggestion
Tip: Store all the App Store Connect variables in the same group so they can be imported to a codemagic.yaml workflow at once. 
```

## Manual code signing

> In order to use manual code signing, Save your **signing certificate**, the **certificate password** (if the certificate is password-protected) and the **provisioning profile** in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. provisioning profiles & .p12 certificate) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to **Environment variables** and decoded during the build.

Reviewer: ```suggestion
In order to use manual code signing, save your **signing certificate**, the **certificate password** (if the certificate is password-protected) and the **provisioning profile** in the **Environment variables** section in Codemagic UI. Click **Secure** to encrypt the values. Note that binary files (i.e. provisioning profiles & .p12 certificate) have to be [`base64 encoded`](../variables/environment-variable-groups/#storing-sensitive-valuesfiles) locally before they can be saved to **Environment variables** and decoded during the build.
```

To set up multiple provisioning profiles, for example, to use app extensions such as [NotificationService](https://developer.apple.com/documentation/usernotifications/unnotificationserviceextension), the easiest option is to add the provisioning profiles to your environment variables with a similar naming convention:
```yaml
environment:
  groups:

>     - provisional_profile

Reviewer: ```suggestion
    - provisioning_profile
```
